lib/Mojo/Reactor/Epoll.pm

```diff
@@ -82,8 +64,9 @@ sub remove {
 		}
 		warn "-- Removed IO watcher for $fd\n" if DEBUG;
 		return !!delete $self->{io}{$fd};
-	} else {
+	} elsif (my $timer = $self->{timers}{$remove}) {
```

Comment:
This leaves no explicit return value if there is no such timer; instead the `$self->remove` line could be guarded on timer existence.

Reply:
Yeah I can do that.